### PR TITLE
 Updating docs to reference NAS #3486

### PR DESCRIPTION
**Pull request by fakela**
#### Summary
This part of docs was updated to show the NAS suggestions under plugins and high availability, by using a bit of what was said under file storage configuration

#### Ticket Link
  Fixes https://github.com/mattermost/docs/issues/3486

### CODE REVIEW

File: source/deployment/cluster.rst [hunk 1]
```diff
@@ -296,7 +296,11 @@ Make sure you have set ``JobSettings.RunScheduler`` to ``true`` in config.json f
...
 
-As of Mattermost 5.14, when you install or upgrade a plugin, it is propagated across the servers in the cluster automatically. 
+As of Mattermost 5.14, when you install or upgrade a plugin, it is propagated across the servers in the cluster automatically.
+
+File storage is assumed to be shared between all the machines that are using services such as NAS or Amazon S3.
+
+If ``"DriverName": "local"`` is used then the directory at ``"FileSettings":`` ``"Directory": "./data/"`` is expected to be a NAS location mapped as a local directory, otherwise high availability will not function correctly and may corrupt your file storage.
 
 Note a slight behaviour change in 5.15:
 When you re-install a plugin in 5.14, the previous **Enabled** or **Disabled** state is retained. As of 5.15, a reinstalled plugin's initial state is **Disabled**.
```
Comment (justinegeffen) [inline source/deployment/cluster.rst:306]:
```suggestion
When you reinstall a plugin in 5.14, the previous **Enabled** or **Disabled** state is retained. As of 5.15, a reinstalled plugin's initial state is **Disabled**.
```

File: source/deployment/cluster.rst [hunk 2]
```diff
@@ -296,7 +296,11 @@ Make sure you have set ``JobSettings.RunScheduler`` to ``true`` in config.json f
 Plugins and High Availability
 ^^^^^^^^^^^^^^^^
 
-As of Mattermost 5.14, when you install or upgrade a plugin, it is propagated across the servers in the cluster automatically. 
+As of Mattermost 5.14, when you install or upgrade a plugin, it is propagated across the servers in the cluster automatically.
+
+File storage is assumed to be shared between all the machines that are using services such as NAS or Amazon S3.
+
+If ``"DriverName": "local"`` is used then the directory at ``"FileSettings":`` ``"Directory": "./data/"`` is expected to be a NAS location mapped as a local directory, otherwise high availability will not function correctly and may corrupt your file storage.
```
Comment (justinegeffen) [inline source/deployment/cluster.rst:303]:
```suggestion
If ``"DriverName": "local"`` is used then the directory at ``"FileSettings":`` ``"Directory": "./data/"`` is expected to be a NAS location mapped as a local directory. If this is not the case high availability will not function correctly and may corrupt your file storage.
```

File: source/deployment/cluster.rst [hunk 3]
```diff
@@ -296,10 +296,14 @@ Make sure you have set ``JobSettings.RunScheduler`` to ``true`` in config.json f
...
 
 Note a slight behaviour change in 5.15:
```
Comment (justinegeffen) [inline source/deployment/cluster.rst:305]:
```suggestion
Note a slight behavior change in 5.15:
```

**Comment by justinegeffen** [comment]
Hi @fakela. Thank you for this PR - it's really helpful and clear. Just a note that I added two, unrelated, commits as I picked them up while reviewing your changes.

File: source/deployment/cluster.rst (hunk 2 above)
Comment (fakela) [reply source/deployment/cluster.rst:303]: This looks good

**Comment by fakela** [comment]
@justinegeffen thanks for reviewing this, the added changes looks good

**Pull request merged**